README.md

```diff
@@ -103,11 +103,22 @@ cd build
 cmake ..
 ```
 
-And that will generate a Visual Studio project inside `build/` ready for you to open in Visual Studio.
+
+And that will generate a Visual Studio project inside `build/` ready for you to open in Visual Studio*.
 
 Alternatively, you can simply open Visual Studio and click `File > Open > CMake...` and just open the `CMakeLists.txt`
 file directly for a slightly different development experience.
 
+#### Visual Studio 2019 users on an x64 Windows Host
```

Comment:
x86_64 or amd64 are the correct names for this architecture

Reply:
I remember in our Systems Architecture class we went over some differences on how x86_64 and amd64 aren't the same thing, so I opted to just rename to 'x86_64'.